### ISSUE TITLE
Error adding a Text Attribute.

### ISSUE THREAD

**Issue by odiea** [opensourcepos/opensourcepos#2476]
The following error is shown when submitting an Item after adding Text to a Text Attribute.
The current Demo does not display this issue so I downloaded the master. It also appends a number after the attribute. I have to submit the Item twice for it to show. This was from my current master so I used the ospos current master.

A PHP Error was encountered
Severity: Notice
Message: Trying to get property 'attribute_id' of non-object
Filename: models/Attribute.php
Line Number: 69
Backtrace:
File: C:\wamp64\www\my-master\application\models\Attribute.php
Line: 69
Function: _error_handler
File: C:\wamp64\www\my-master\application\models\Attribute.php
Line: 530
Function: value_exists
File: C:\wamp64\www\my-master\application\controllers\Items.php
Line: 625
Function: save_value
File: C:\wamp64\www\my-master\public\index.php
Line: 315
Function: require_once

{"success":true,"message":"You have successfully updated item socks","id":"3"}

Golden Retriever1

OSPOS Installation Info: 3.3.0 - dev
Language Code: en-US
Extensions & Modules:
» GD: Enabled ✓
» BC Math: Enabled ✓
» INTL: Enabled ✓
» OpenSSL: Enabled ✓
» MBString: Enabled ✓
» Curl: Enabled ✓

User Settings:
.Browser: Chrome
.Server Software: Apache/2.4.33 (Win64) OpenSSL/1.1.0g PHP/7.3.2
.PHP Version: 7.3.2
.DB Version: 5.7.21
.Server Port: 80
.OS: Windows NT PAVILION23 10.0 build 17763 (Windows 10) AMD64

File Permissions:
» [application/logs:] - 0777 | Writable ✓ 
» [public/uploads:] - 0777 | Writable ✓ 
» [public/uploads/item_pics:] - 0777 | Writable ✓ 
» [import_customers.csv:] - 0666 | Writable ✓

**Comment by andriux1990** [comment]
I also did the tests and it's true. After the added text, add two numbers (55)

Why is this error?

**Comment by jekkos** [comment]
This error seems to be related to bootstrap tables itself. It tries to toggle a column visibility for an attribute id that is not in the table (or so). Can you try to remove localStorage and see if that helps?

Type as follows in the browser javascript console, after heading to the ospos domain you are testing on
`delete localStorage

**Comment by jekkos** [comment]
I tried to upgrade to a newer bootstrap tables version today, it seems that the error is not there with 1.13.4. They are at 1.15.2 already, but it uses ECMAscript 6 and bootstrap 4. So we're lagging behind a bit there.
1.13.3 does seem to work except for a last CSS issue that refuses to inline the bootstrap table css.

**Comment by jekkos** [comment]
Pushed another fix that should remove the additional characters as @andriux1990 reported.

**Comment by jekkos** [comment]
Did anyone had a chance to check this one?

**Comment by daN4cat** [comment]
@odiea can you please test and confirm?

**Comment by odiea** [comment]
I downloaded the latest and greatest.
When I first tried to Login this showed.
ERROR - 2019-07-06 04:22:29 --> Query error: Duplicate entry '196uv8866e0f51sujveqdl957mu13j51' for key 'PRIMARY' - Invalid query: INSERT INTO `ospos_sessions` (`id`, `ip_address`, `timestamp`, `data`) VALUES ('196uv8866e0f51sujveqdl957mu13j51', '::1', 1562408549, '__ci_last_regenerate|i:1562408549;person_id|s:1:\"1\";')
ERROR - 2019-07-06 04:22:29 --> Severity: Warning --> Unknown: Cannot call session save handler in a recursive manner Unknown 0
ERROR - 2019-07-06 04:22:29 --> Severity: Warning --> Unknown: Failed to write session data using user defined save handler. (session.save_path: c:/wamp64/tmp) Unknown 0
ERROR - 2019-07-06 10:24:33 --> 404 Page Not Found: Uploads/company_logo.png
I changed db and it logged in ok.
I went to add a text Description to an Item.
On the first submit I got the following error with no form submission.
ERROR - 2019-07-06 04:29:06 --> Severity: Notice --> Trying to get property 'attribute_id' of non-object C:\wamp64\www\7619\application\models\Attribute.php 69
Second submit all worked good and Items table shows the new Description without any numbers following the Text.
![2019-07-06 04_29_24-Window](https://user-images.githubusercontent.com/10962177/60755045-8f0f2500-9fa7-11e9-9411-54833327163e.png)

**Comment by daN4cat** [comment]
Is this still an issue?

**Comment by odiea** [comment]
I downloaded the current bintray Master. I created a new db for this test. I created an Item. I went to attributes and created a text attribute to show in all 3. Went back to Items. No Items displayed in the table. Went back to Attributes. Removed the check mark to display in Items. Went back to Items. The table showed. I added an item text attribute . went ok. Re enabled text attribute to show in Items and Table loaded properly. I put it in production mode and had no issue. I guess it just shows in development mode.
